src/components/textarea.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import TextareaAutosize from 'react-textarea-autosize';
+import classNames from 'classnames';
+
+export interface TextareaProps {
+    /** Textarea id. Must be unique */
+    id: string;
+    /** The value of the checkbox */
+    value: string;
+    /** Called when the checkbox is clicked */
+    onChangeHandler: (value: string) => void;
+    /** When true the error state will be displayed */
+    error?: boolean;
+    /** Whether the textarea should grow horizontally with user input */
+    autoGrow?: boolean;
+    /** html placeholder text */
+    placeholder?: string;
+    /** Whether the textarea should be auto-focused */
+    autofocus?: boolean;
```

Comment:
autoGrow is camelcased but autofocus is not?

Reply:
Autofocus is one word